learn/security/master_api_keys.md

```diff
@@ -158,34 +162,40 @@ We can query our instance to confirm which active keys can search our `patient_m
 {
   "results": [
     {
-      "description": "Default Search API Key (Use it to search from the frontend code)",
-      "key": "0a6e572506c52ab0bd6195921575d23092b7f0c284ab4ac86d12346c33057f99",
+      "name": "Default Search API Key",
+      "description": "Default Search API Key",
+      "key": "d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4",
+      "uid":"74c9c733-3368-4738-bbe5-1d18a5fecb37",
       "actions": [
-          "search"
+        "search"
       ],
       "indexes": [
-          "doctors"
+        "*"
       ],
       "expiresAt": null,
-      "createdAt": "2021-08-11T10:00:00Z",
-      "updatedAt": "2021-08-11T10:00:00Z"
+      "createdAt": "2022-01-01T10:00:00Z",
+      "updatedAt": "2022-01-01T10:00:00Z"
     },
     {
+      "name": "Default Admin API Key",
       "description": "Default Admin API Key (Use it for all other operations. Caution! Do not share it on the client side)",
```

Comment:
I think the description for the default API keys changed _slightly_ (ref: https://github.com/meilisearch/specifications/pull/148/files#diff-efbeaa05fe855150edfa47dc218874a3d03dda4000c79375238288abd551694aR101).

```suggestion
      "description": "Use it for all other than search operations. Caution! Do not expose it on a public frontend", 
```